### PR TITLE
add: enable to fetch capital gains available on Mutual Funds

### PR DESCRIPTION
**Pull request by florspag**

### CODE REVIEW

File: src/quotes.rs [hunk 1]
```diff
@@ -245,3 +262,12 @@ pub struct Dividend {
...
 }
+
+/// This structure simply models a capital gain which has been recorded.
+#[derive(Deserialize, Debug, Clone)]
+pub struct CapitalGain {
+    /// This is the price of the dividend
+    pub amount: f64,
+    /// This is the ex-dividend date
+    pub date: u64,
+}
```
Comment (xemwebe) [inline src/quotes.rs]: add new line here

**Comment by xemwebe** [comment]
Thanks for providing this PR!
Could you also add a unit test for this new feature?

File: src/quotes.rs (hunk 1 above)
Comment (florspag) [reply src/quotes.rs]: sure it's done

**Comment by florspag** [comment]
Yes i've added unit test for this new feature.
Regards,

**Pull request merged**